<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EventReceiverWindow</class>
 <widget class="QMainWindow" name="EventReceiverWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>540</width>
    <height>307</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Event receiver</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <property name="horizontalSpacing">
     <number>12</number>
    </property>
    <item row="0" column="0" colspan="4">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="spacing">
         <number>5</number>
        </property>
        <item>
         <widget class="QESimpleShape" name="ledEnable">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>16</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16</width>
            <height>20</height>
           </size>
          </property>
          <property name="shape">
           <enum>QSimpleShape::circle</enum>
          </property>
          <property name="colour0" stdset="0">
           <color>
            <red>20</red>
            <green>50</green>
            <blue>20</blue>
           </color>
          </property>
          <property name="colour1" stdset="0">
           <color>
            <red>5</red>
            <green>240</green>
            <blue>5</blue>
           </color>
          </property>
          <property name="variable" stdset="0">
           <string>$(prefix):isEnabled</string>
          </property>
          <property name="variableSubstitutions" stdset="0">
           <string/>
          </property>
          <property name="displayAlarmStateOption" stdset="0">
           <enum>QE::Never</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lblEnable">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>60</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>60</width>
            <height>20</height>
           </size>
          </property>
          <property name="text">
           <string>Enable</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QEPushButton" name="btnOFF">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
          <property name="text">
           <string>OFF</string>
          </property>
          <property name="variable" stdset="0">
           <string>$(prefix):enable</string>
          </property>
          <property name="clickText">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QEPushButton" name="btnON">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
          <property name="text">
           <string>ON</string>
          </property>
          <property name="variable" stdset="0">
           <string>$(prefix):enable</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>47</width>
          <height>17</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>5</number>
        </property>
        <item>
         <widget class="QESimpleShape" name="ledRXViolation">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>16</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16</width>
            <height>20</height>
           </size>
          </property>
          <property name="ledType">
           <enum>QSimpleShape::Error</enum>
          </property>
          <property name="shape">
           <enum>QSimpleShape::circle</enum>
          </property>
          <property name="colour0" stdset="0">
           <color>
            <red>5</red>
            <green>240</green>
            <blue>5</blue>
           </color>
          </property>
          <property name="colour1" stdset="0">
           <color>
            <red>240</red>
            <green>5</green>
            <blue>5</blue>
           </color>
          </property>
          <property name="variable" stdset="0">
           <string>$(prefix):isRxViolation</string>
          </property>
          <property name="variableSubstitutions" stdset="0">
           <string/>
          </property>
          <property name="displayAlarmStateOption" stdset="0">
           <enum>QE::Never</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lblRX">
          <property name="minimumSize">
           <size>
            <width>90</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>90</width>
            <height>20</height>
           </size>
          </property>
          <property name="text">
           <string>RX Violation</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QEPushButton" name="btnRXReset">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>71</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>71</width>
            <height>20</height>
           </size>
          </property>
          <property name="text">
           <string>Reset</string>
          </property>
          <property name="variable" stdset="0">
           <string>$(prefix):resetRxViolation</string>
          </property>
          <property name="variableSubstitutions" stdset="0">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>326</width>
          <height>17</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item row="1" column="0" colspan="4">
     <widget class="QGroupBox" name="groupBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QGroupBox {
     border: 1px solid black;
     border-radius: 5px;
     margin-top: 0.5em;
 }
 
 QGroupBox::title {
     subcontrol-origin: margin;
     left: 10px;
     padding: 0 3px 0 3px;
 }</string>
      </property>
      <property name="title">
       <string>Pulsers Outputs</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QVBoxLayout" name="ltPulsers">
         <property name="spacing">
          <number>2</number>
         </property>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="0" colspan="4">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QGroupBox {
     border: 1px solid black;
     border-radius: 5px;
     margin-top: 0.5em;
 }
 
 QGroupBox::title {
     subcontrol-origin: margin;
     left: 10px;
     padding: 0 3px 0 3px;
 }</string>
      </property>
      <property name="title">
       <string>DelayedPulsers Outputs</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QVBoxLayout" name="ltDelayed">
         <property name="spacing">
          <number>2</number>
         </property>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QGroupBox" name="groupBox_4">
      <property name="styleSheet">
       <string notr="true">QGroupBox {
     border: 1px solid black;
     border-radius: 5px;
     margin-top: 0.5em;
 }
 
 QGroupBox::title {
     subcontrol-origin: margin;
     left: 10px;
     padding: 0 3px 0 3px;
 }</string>
      </property>
      <property name="title">
       <string>TTL Outputs</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="ltTTL">
         <property name="spacing">
          <number>0</number>
         </property>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QGroupBox" name="groupBox_3">
      <property name="styleSheet">
       <string notr="true">QGroupBox {
     border: 1px solid black;
     border-radius: 5px;
     margin-top: 0.5em;
 }
 
 QGroupBox::title {
     subcontrol-origin: margin;
     left: 10px;
     padding: 0 3px 0 3px;
 }</string>
      </property>
      <property name="title">
       <string>Universal Outputs</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QVBoxLayout" name="ltUniversal">
         <property name="spacing">
          <number>0</number>
         </property>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QGroupBox" name="groupBox_5">
      <property name="styleSheet">
       <string notr="true">QGroupBox {
     border: 1px solid black;
     border-radius: 5px;
     margin-top: 0.5em;
 }
 
 QGroupBox::title {
     subcontrol-origin: margin;
     left: 10px;
     padding: 0 3px 0 3px;
 }</string>
      </property>
      <property name="title">
       <string>CML Outputs</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <layout class="QVBoxLayout" name="ltCML">
         <property name="spacing">
          <number>0</number>
         </property>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QGroupBox" name="groupBox_6">
      <property name="styleSheet">
       <string notr="true">QGroupBox {
     border: 1px solid black;
     border-radius: 5px;
     margin-top: 0.5em;
 }
 
 QGroupBox::title {
     subcontrol-origin: margin;
     left: 10px;
     padding: 0 3px 0 3px;
 }</string>
      </property>
      <property name="title">
       <string>Prescalers</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QVBoxLayout" name="ltPrescalers">
         <property name="spacing">
          <number>0</number>
         </property>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="4" column="0" colspan="4">
     <widget class="QGroupBox" name="GBClose">
      <property name="styleSheet">
       <string notr="true">QGroupBox {
 	background-color: rgb(255, 255, 255);
     border: 1px solid black;
     border-radius: 5px;
     margin-top: 0.5em;
 }
 
 QGroupBox::title {
     subcontrol-origin: margin;
     left: 10px;
     padding: 0 3px 0 3px;
 }</string>
      </property>
      <property name="title">
       <string/>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>413</width>
           <height>27</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="btnClose">
         <property name="text">
          <string>Close</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QEPushButton</class>
   <extends>QPushButton</extends>
   <header>QEPushButton.h</header>
  </customwidget>
  <customwidget>
   <class>QESimpleShape</class>
   <extends>QSimpleShape</extends>
   <header>QESimpleShape.h</header>
  </customwidget>
  <customwidget>
   <class>QSimpleShape</class>
   <extends>QWidget</extends>
   <header>QSimpleShape.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
